### ISSUE TITLE
[508] Installer page headers are not called out by JAWS.

### ISSUE THREAD

**Issue by glassfishrobot** [javaee/glassfish#14222]
build used : GF V3.1 b25 promoted.
Os: windows XP.
Installer bundle : latest-ogs-windows.exe

Jaws reader does not read out the Page Headers in the screens, like
"Introduction",Installation type, Install directory etc.
Jaws reads all white frame page texts and all buttons,radio buttons, and check
boxes. but not page headers.
This should be fixed.
#### Environment
Operating System: All
Platform: All
#### Affected Versions
[3.1]

**Comment by glassfishrobot** [comment]
**Closing this as this issue is migrated to** https://github.com/eclipse-ee4j/glassfish/issues/14222
